### ISSUE TITLE
Test Run Get Trace unit test fails

### ISSUE THREAD

**Issue by janvanrijn** [openml/openml-python#565]
```
Error
Traceback (most recent call last):
  File "/home/janvanrijn/anaconda3/envs/openml-python/lib/python3.6/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/home/janvanrijn/anaconda3/envs/openml-python/lib/python3.6/unittest/case.py", line 605, in run
    testMethod()
  File "/home/janvanrijn/projects/openml-python/tests/test_runs/test_run_functions.py", line 610, in test_get_run_trace
    trace = openml.runs.functions._create_trace_from_arff(
AttributeError: module 'openml.runs.functions' has no attribute '_create_trace_from_arff'
```

Makes sense.

**Comment by janvanrijn** [comment]
Come to think of it, I am a little bit surprised that this error did not manifest itself before. I reset the test server in the meantime, but I could not think of a reason why it didn't fail before.

**Comment by janvanrijn** [comment]
Got it, it's because the try/catch around `avoid_duplicate_runs=True`

Although this is hard to test, I think just fixing this is fine.

**Comment by mfeurer** [comment]
Where's this try/except statement?

**Comment by mfeurer** [comment]
Closed by merging #566.
